src/pages/CommitDetailPage/CommitBundleAnalysis/CommitBundleAnalysis.tsx

```diff
@@ -0,0 +1,25 @@
+import { lazy, Suspense } from 'react'
+
+import Spinner from 'ui/Spinner'
+
+const CommitBundleAnalysisTable = lazy(
+  () => import('./CommitBundleAnalysisTable')
+)
+
+const Loader = () => (
+  <div className="flex flex-1 justify-center">
+    <Spinner />
+  </div>
+)
+
+const CommitBundleAnalysis: React.FC = () => {
+  return (
+    <>
```

Comment:
is there a need for this extra fragment?

Reply:
This is really just a placeholder component until it's fully fleshed out, just gonna leave as is.

Reply:
do you mean we'll add a component inside of this fragment outside of suspense?

Reply:
This component is going to drastically change in the very very near future.

Reply:
i still think we should remove the extra fragment though as we're viewing the current code, approving to unblock